cmd/telemetrygen/internal/traces/traces.go

```diff
@@ -115,6 +116,14 @@ func Run(c *Config, logger *zap.Logger) error {
 		logger.Info("generation of traces is limited", zap.Float64("per-second", float64(limit)))
 	}
 
+	var statusCode codes.Code
+	if c.StatusCode == "" {
+		statusCode = codes.Unset
+	} else {
+		if err := statusCode.UnmarshalJSON([]byte(c.StatusCode)); err != nil {
+			return fmt.Errorf("`status-code` must be one of (Unset, Error, Ok) or (0, 1, 2)")
```

Comment:
```suggestion
			return fmt.Errorf("expected `status-code` to be one of (Unset, Error, Ok) or (0, 1, 2), got %q instead", c.StatusCode)
```

Reply:
Thanks for your suggestion, I have made the change.